extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -1053,3 +1053,12 @@ const WarningMessageTokenDetails = ({
     </div>
   );
 };
+
+export const MaliciousDomainWarning = ({ message }: { message: string }) => (
+  <div className="MaliciousDomainWarning">
+    <div className="Icon">
+      <Icon.Warning className="WarningMessage__icon" />
```

Comment:
This icon is different from the one used in the mocks, I think this is an SDS v1 vs v2 difference.